src/components/organisms/about/about.html

```diff
@@ -0,0 +1,19 @@
+<div class="about">

```

Comment:
```suggestion
<div class="about container">
```
現在横幅いっぱいに広がってしまっているので、bootstrapのcontainerを使って余白をいい感じに調整してください。